Flashlight-VNC/src/com/flashlight/vnc/VNCClient.as

```diff
@@ -103,7 +103,7 @@ package com.flashlight.vnc
 		[Bindable] public var securityPort:int = 0;
 		[Bindable] public var shareConnection:Boolean = true;
 		[Bindable] public var password:String;
-		[Bindable] public var pastePauseDelay:int = 50;
+		[Bindable] public var pastePauseDelay:int = 25;
```

Comment:
* What environments did you test this in?
* Does this change correctly paste long strings with higher `pastePauseDelay`s in Safari + IOS (150 IIRC)? The IOS paste problem only exhibits when pasting into text boxes in the simulator e.g. the google search box
* Can we decrease the `pastePauseDelay` in Safari/IOS with this change?